README.md

```diff
@@ -45,6 +45,25 @@ to your Prometheus server, and the files in `dashboards_out` need to be imported
 into you Grafana server.  The exact details will depending on how you deploy your
 monitoring stack to Kubernetes.
 
+### Dashboards for Windows Nodes
+There are separate dashboards for windows resources.
+1) Compute Resources / Cluster(Windows)
+2) Compute Resources / Namespace(Windows)
+3) Compute Resources / Pod(Windows)
+4) USE Method / Cluster(Windows)
+5) USE Method / Node(Windows)
+
+These dashboards are based on metrics populated by wmi_exporter(https://github.com/martinlindhe/wmi_exporter) from each Windows node.
+
+Steps to configure wmi_exporter
+1) Download the latest version of wmi_exporter from release page(https://github.com/martinlindhe/wmi_exporter/releases/)
```

Comment:
We should specify that this requires at least v0.7.0 or higher

Reply:
Updated.